packages/instantsearch.css/astro.config.mjs

```diff
@@ -0,0 +1,5 @@
+import { defineConfig } from "astro/config";
```

Comment:
could be done in a separate PR, but instantsearch.css should be only the css files, and the specs should be its own package (or folder in website)

Reply:
The PR is still a WIP.